Assets/MixedRealityToolkit-SDK/Features/SpatialAwareness/System/WindowsMixedRealitySpatialObserver.cs

```diff
@@ -121,20 +106,19 @@ public override void Destroy()
         /// one as a spare for use in outstanding mesh requests. That way, we'll have fewer
         /// game object create/destroy cycles, which should help performance.
         /// </summary>
-        protected SpatialMeshObject? spareMeshObject = null;
+        private SpatialMeshObject? spareMeshObject = null;
 
         /// <summary>
         /// The time at which the surface observer was last asked for updated data.
         /// </summary>
         private float lastUpdated = 0;
-#endif // UNITY_WSA
 
         /// <inheritdoc />
         public override IDictionary<int, GameObject> Meshes
         {
             get
             {
-                Dictionary<int, GameObject> meshes = new Dictionary<int, GameObject>();
+                var meshes = new Dictionary<int, GameObject>();
```

Comment:
var makes me sad :)

Reply:
Lol, but you already know the type you're creating on the right :p

Reply:
Yeah, I know... old habits die hard :)